package.json

```diff
@@ -11,7 +11,8 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "grunt test-dev"
+    "ci": "karma start karma.conf.js --single-run",
+    "test": "karma start karma.conf.js"
```

Comment:
I think test should be our single run. Maybe that is an alignment with travis and other CI tools that default to npm test but I think that also colors what `npm test` means to me. A single run through the tests.